analyses/README.md

```diff
@@ -53,6 +53,7 @@ Note that _nearly all_ modules use the harmonized clinical data file (`pbta-hist
 | [`selection-strategy-comparison`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/selection-strategy-comparison) | `pbta-gene-expression-rsem-fpkm.polya.rds` <br> `pbta-gene-expression-rsem-fpkm.stranded.rds` | *Deprecated*; Comparison of RNA-seq data from different selection strategies | N/A
 | [`sex-prediction-from-RNASeq`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/sex-prediction-from-RNASeq) | `pbta-gene-expression-kallisto.stranded.rds` <br> `pbta-histologies.tsv` | *In progress*; predicts genetic sex using RNA-seq data ([#84](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/6)) | N/A
 | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) | `pbta-snv-lancet.vep.maf.gz` <br> `pbta-snv-mutect2.vep.maf.gz` <br> `pbta-snv-strelka2.vep.maf.gz` <br> `pbta-snv-vardict.vep.maf.gz` <br> `tcga-snv-lancet.vep.maf.gz` <br> `tcga-snv-mutect2.vep.maf.gz` <br> `tcga-snv-strelka2.vep.maf.gz` | Generates consensus SNV and indel calls for PBTA and TCGA data; calculates tumor mutation burden using the consensus calls | `results/consensus/pbta-snv-consensus-mutation.maf.tsv.gz` (included in data download; too large for tracking via GitHub) <br> `results/consensus/pbta-snv-consensus-mutation-tmb-all.tsv` <br> `results/consensus/pbta-snv-consensus-mutation-tmb-coding.tsv`(included in data download; too large for tracking via GitHub) <br> `results/consensus/tcga-snv-consensus-mutation.maf.tsv.gz` <br> `results/consensus/tcga-snv-mutation-tmb.tsv` <br> `results/consensus/tcga-snv-mutation-tmb-coding.tsv`
+| [`snv-frequencies`](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/tree/dev/analyses/snv-frequencies) | `histologies.tsv` <br> `snv-consensus-plus-hotspots.maf.tsv.gz` <br> `independent-samples/results/independent-specimens.wgs.primary.tsv` <br> `independent-samples/results/independent-specimens.wgs.relapse.tsv` | Annotate SNV table with mutation frequencies | N/A
```

Comment:
Just a note that you will want to add the WXS data with v7 as well

Reply:
Thank you for the note. This is good to know! I will use `wgswxspanel` independent sample lists in the upcoming snv v7 update PR.